resilience4j-circuitbreaker/src/main/java/io/github/resilience4j/circuitbreaker/CircuitBreakerConfig.java

```diff
@@ -174,6 +181,7 @@ public Builder(CircuitBreakerConfig baseConfig) {
             this.automaticTransitionFromOpenToHalfOpenEnabled = baseConfig.automaticTransitionFromOpenToHalfOpenEnabled;
             this.slowCallRateThreshold = baseConfig.slowCallRateThreshold;
             this.slowCallDurationThreshold = baseConfig.slowCallDurationThreshold;
+            this.disableStackTraces = baseConfig.disableStackTraces;
```

Comment:
Can we rename it to `writableStackTraceEnabled` and set it to `true` by default in order to be backward compatible.